src/main/webapp/js/instructor.js

```diff
@@ -14,6 +14,11 @@ $(document).ready(function() {
     
     // bind the show picture onhover events
     bindStudentPhotoHoverLink('.profile-pic-icon-hover');
+    
+    // Formatting searchKey for highlight searchKey plugin ( ['string1','string2',...] )
```

Comment:
Perhaps, one way to make this more safe (for other contributors) and consistent throughout the app, the following can be done:

1. Abstract the whole thing into a function
2. Move the function into a common file ```common.js``` comes into mind
3. Call the function in the files that needs this functionality

```
// Some comment
function highlightSearchKeys() {
  var searchKey = $('#searchBox').val();
  var splitSearchKey = searchKey.split(' ');
  $('.panel-body').highlight(splitSearchKey);
}
```

What doing the above achieves:

1. Reduces variable namespace pollution in the function scope of ```$(document).ready(function() {...``` which could be accidentally mangled by other contributors - this ties in with the principle of keeping things as safe as possible for future contributions, keep variables that "others" do not need to know about or have access to in their own "private namespace" so that there is no chance that they will be mis-used
2. In the future if we need to highlight things from say ```$('.some-special-content')``` we can just add it in the function and functionality will be consistent throughout the app, contributors will not miss any part (for instance another contributor who doesn't know about this feature now might not change the JavaScript file in the admin search page) - this has got to do with the principle of preventing regressions and preventing functionality divergence
3. Reduce code duplication - DRY principle

Note that if we take up the approach above, we can also refactor the admin part to use the same code - rejoice!